### ISSUE TITLE
Can't copy-paste text from log

### ISSUE THREAD

**Issue by sqshemet** [mbari-org/dash5#287]
**Describe the bug**
Nothing happens when I click and drag to highlight log text to copy it. This is really important for e.g. asking questions in Slack

**To Reproduce**
1. Go to Log (either in sidebar or fullscreened)
2. Click anywhere and hold
3. Move mouse to anywhere
4. Nothing happens :(

**Expected behavior**
Log text should be highlighted and copyable

**Desktop (please complete the following information):**
 - OS: Ubuntu 20.04
 - Browser: Chrome
 - Version 109.0.5414.119 (Official Build) (64-bit)

**Comment by ZackLyon** [comment]
@ksalamy - Is this closed by #435 ?

**Comment by sqshemet** [comment]
It seems like the current behavior is that clicking on a single log event copies it to the clipboard? e.g.
```Critical
Time: 11:58
Date: Today
Direction: Download

[CBIT] Communications Fault in component: DockingStepper
```

I think we at least need to be able to copy multiple log events at once. In my opinion it'd be nice to be able to select text "normally" and therefore be able to copy partial log events, (e.g. "Communications fault in component: DockingStepper") but I don't think it's strictly necessary. I'd also vote for not including additional headers and not making the direction copyable. I'd naively expect the example event to copy like "Critical 11:58 Today [CBIT] Communications Fault in component: DockingStepper"

Personally I'd *want* it to copy something like "2025-10-02 11:58:ss Critical [CBIT] Communications Fault in component: DockingStepper", but I might be missing earlier discussion on how to format the new log (@heavyB ?)

**Comment by heavyB** [comment]
Agreed we often copy multiple events when responding to questions with PICs. It seems cumbersome to have to concat

**Comment by jimjeffers** [comment]
Do you want the ability to explicitly select certain logs or just a button to download a txt file of all of the logs currently loaded into the component?

**Comment by sqshemet** [comment]
I think we need the ability to explicitly select certain logs. IMO it's fine if that's limited to adjacent logs within the current display, I don't think we need arbitrary selection. Just something that is as capable as a "normal" copy-paste.

**Comment by ZackLyon** [comment]
Closed via #504
